config/mb_config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "default":
+    {
+      "port": "4733"
+    }
+  ,
+  "mongo":
+    {
+      "development" : "mongodb://localhost/mb-logging",
+      "production" : "mongodb://mongo-apps,mongo4-aws,mongo5-aws:27017/mb-logging/?replicaSet=rs1"
+    }
+}
```

Comment:
Let's leave it here for now. I'm working on this issue now in vagrant, maybe I'll figure out something better.